curriculum/challenges/_meta/lecture-working-with-links/meta.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Working with Links",
+  "blockType": "lecture",
+  "isUpcomingChange": true,
+  "dashedName": "lecture-working-with-links",
+  "order": 8,
+  "superBlock": "front-end-development",
+  "challengeOrder": [
+    {
+      "id": "6716744f7245947a3dd60009",
+      "title": "What Are the Different Target Attribute Types, and How Do They Work?"
+    },
+    {
+      "id": "671682dd88e461a8e2620f38",
+      "title": "What Is the Difference Between Absolute and Relative Paths?"
+    },
+    {
+      "id": "6716830dbaf95da9564f2e3b",
+      "title": "What Is the Difference Between Slashes, a Single Dot, and Double Dot in Path Syntax?"
+    },
```

Comment:
I believe these too are going to be merged into one script because of repetitive text in both scripts.
So you should reach out to Naomi about that

Reply:
Would we be fine to merge it like this and come back to it? Just wanting to get this in for the demo. Maybe @naomi-lgbt can weigh in.

Reply:
Yeah, if this is just for the demo, then we can merge in as is and then revisit this later. 
I don't remember where Naomi and Ania landed on this. Whether they decided to keep both videos and make updates or merge it into one script.